<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadManagerWidget</class>
 <widget class="QWidget" name="DownloadManagerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadManagerWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelURL">
         <property name="text">
          <string>URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditURL"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QToolButton" name="buttonAuthenticationServer">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/ico/KeyChainAccess.ico</normaloff>:/ico/KeyChainAccess.ico</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAuthenticationServer">
     <property name="title">
      <string>Server Anthentication</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditUsername"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxServerPort">
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>21</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Download Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboBoxOptionsWhenCompleted">
        <item>
         <property name="text">
          <string>Do nothing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Suspend</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Make a sound</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Turn off Computer</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSaveLocation"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="buttonChooseSavedLocation">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/ico/QuickTime.ico</normaloff>:/ico/QuickTime.ico</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>When downloads complete: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Save file to:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBarDowloadedSize">
     <property name="maximum">
      <number>2048</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Downloaded: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDownloadedSize">
       <property name="text">
        <string>0/0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>468</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonDownload">
       <property name="text">
        <string>Download</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="buttonOpenFileLocation">
       <property name="text">
        <string>Open Location</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonAuthenticationServer</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBoxAuthenticationServer</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
